### PR TITLE
Add missing Archetypes from DA, QftFF, NotGD, OoA, and BotD to Archetype Journal

### PR DESCRIPTION
**Pull request by dogstarrb**

### CODE REVIEW

File: packs/data/journals.db/archetypes.json [hunk 1]
```diff
@@ -13,8 +13,9 @@
...
...m your starting position. This movement doesn't trigger reactions. You can't move farther than your Speed, and you must end your movement adjacent to the enemy whose space you moved through. After moving, you make a melee Strike against the enemy whose space you moved through, and the enemy is flat-footed against that Strike.</p><p><strong>Success</strong> As critical success, but the enemy isn't flat-footed against the Strike.</p><p><strong>Failure</strong> You remain in your original space but can still Strike.</p><p><strong>Critical Failure</strong> No effect.</p><h2>@Compendium[pf2e.feats-srd.Tumbling Opportunist]{Tumbling Opportunist} Feat 10</h2><p><strong>Prerequisites</strong> Acrobat Dedication</p><p><strong>Frequency</strong> once per minute</p><p><strong>Requirements</strong> Your most recent action was to Tumble Through or Tumbling Strike, and you successfully moved through an enemy's space.</p><p>You use a burst of stamina to perform a breathtaking feat of Acrobatics as you speed through a foe's space, leaving your foe lying flat on their back. You attempt to Trip the enemy whose space you moved through. You can use Acrobatics instead of Athletics for this check.</p>",
```
Comment (TikaelSol) [inline packs/data/journals.db/archetypes.json:16]: It looks like the only thing that is different on these is the new line characters are removed. We can exclude these changes from this and the changes to other existing items in the journal.

File: packs/data/journals.db/archetypes.json [hunk 2]
```diff
@@ -2553,6 +2561,423 @@
...
...p></p><h2>@UUID[Compendium.pf2e.feats-srd.JEzjyNbpsh05iymG]{Prepare Papers} Feat 7</h2><p><strong>Prerequisites </strong>Alkenstar Agent Dedication, master in Society</p><hr /><p>You're practiced at creating forgeries that establish your outlaw credentials or augment your legal authority. You can @Compendium[pf2e.actionspf2e.Create Forgery]{Create Forgery} as an exploration activity. It takes you about 1 hour per page of the document that you're forging. This time is reduced to 10 minutes per page if you're legendary in Society.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.qbFRPTHP96Q9kGpk]{Flash Your Badge} Level 10</h2><p><strong>Prerequisites </strong>Alkenstar Agent Dedication, master in Intimidation</p><hr /><p>You make a fearsome show of your authority. Roll Intimidation checks to @Compendium[pf2e.actionspf2e.Demoralize]{Demoralize} each creature in a @Template[type:cone|distance:30]. When you do so, Demoralize gains the visual trait, and creatures are affected if they can see you. If a target creature hasn't broken a law in the past week (as determined by the GM), the result of your check against that creature is one degree of success worse than the result you rolled.</p>",
```
Comment (TikaelSol) [inline packs/data/journals.db/archetypes.json:2573]: All of these have some changes that need made. `<span id=\"ctl00_RadDrawer1_Content_MainContent_DetailedOutput\">` and the connected `</span>` tag need to go.

The style tags and other stuff like

```
<p style=\"box-sizing:border-box;user-select:text;margin:0.5em 0px;min-height:unset;font-weight:500;color:rgb(25, 24, 19);font-family:Roboto, sans-serif;font-size:14px;font-style:normal;font-variant-ligatures:normal;font-variant-caps:normal;letter-spacing:normal;orphans:2;text-align:start;text-indent:0px;text-transform:none;white-space:normal;widows:2;word-spacing:0px;-webkit-text-stroke-width:0px;text-decoration-thickness:initial;text-decoration-style:initial;text-decoration-color:initial\"><span style=\"font-family:Roboto, sans-serif\">
```

Also need to go. Just remove the style from the paragraph tags and delete the span tags entirely. The only span tags we use are to float content to the right and the action icons.

File: packs/data/journals.db/archetypes.json [hunk 3]
```diff
@@ -2553,6 +2561,423 @@
...
...flection for 1 minute. Once on each of its turns, the target can spend 1 action to attempt another Will save to escape. If the save succeeds, the effect ends early.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.kC92uxCvrxDkrCpO]{Borrow Memories} Feat 14</h2><p><strong>Prerequisites </strong>Alter Ego Dedication</p><hr /><p>Knowing how someone thinks makes it much easier to know where they'll be, or even replace them altogether. You can cast <em>@Compendium[pf2e.spells-srd.Mind Probe]{Mind Probe}</em> as an innate occult spell once per day, using your class DC or spell DC, whichever is higher.</p><p>If you cast <em>mind probe</em> on a target you're studying for the purpose of Assuming their Role, and the target fails their save, you can delay the spell from coming into effect until you Assume their Role. When you use the spell this way, you can Sustain it up to 10 times during its duration, asking a question each time. The spell lasts until you ask all 10 questions or are no longer Assuming the Role, whichever comes first. You don't need to Sustain the Spell to extend its duration-only when asking a question-nor do you need to be in range or have line of effect to Sustain it.</p>",
```
Comment (TikaelSol) [inline packs/data/journals.db/archetypes.json:2595]: We don't use `<br>` tags. Do line breaks with new paragraphs `<p>First paragraph.</p><p>Second paragraph.</p>`

Also looks like there's some blank paragraphs in here `</p><p></p><h2>` we can ditch those.

**Comment by dogstarrb** [comment]
I'm not sure where anything html was coming from, since other than the sidebar information, I did all edits in the Foundry text editor. I'll go back and try to clean them up, though.

**Comment by TikaelSol** [comment]
If you paste into the editor it adds the source formatting, or attempts to. Control shift v pastes without formatting.

**Pull request merged**